package.json

```diff
@@ -4,10 +4,13 @@
   "description": "An HTMLElement extension that provides a simple setup for Web Components using React.",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
```

Comment:
Added this so we can delete the .npmignore (which caused npm to not consider .gitignore).